### PR TITLE
Merge with tootsuite @ 57b503d4ef0754e4651eae26b628cba60e1ab4e9

### PR DESCRIPTION
**Pull request by yipdw**
As far as I can tell, the big change this brings in is the ability to hide your follower/following lists in both HTML and JSON.

### CODE REVIEW

**Comment by yipdw** [comment]
The Ruby 2.5 build looks unhappy: https://travis-ci.org/glitch-soc/mastodon/jobs/380679074#L798

**Comment by yipdw** [comment]
Ah thanks, I missed that.

File: app/lib/user_settings_decorator.rb [hunk 1]
```diff
@@ -84,12 +85,21 @@ def noindex_preference
...
   def skin_preference
     settings['setting_skin']
+=======
+  def hide_network_preference
+    boolean_cast_setting 'setting_hide_network'
+  end
+
+  def theme_preference
+    settings['setting_theme']
+>>>>>>> origin/master
```
Comment (ThibG) [inline app/lib/user_settings_decorator.rb]: Oops merge conflict

**Comment by ThibG** [comment]
Could you amend the merge commit instead? I find that far more readable (but you may disagree I guess)

**Comment by yipdw** [comment]
Yep.  Merge amended; new `merge-upstream` HEAD is 399af266e57f68377194922d1141ad675d2f4b0e.

**Comment by ThibG** [comment]
Not sure what's happening with continuous integration here :/

**Comment by rhaamo** [comment]
Same ````require': cannot load such file -- bundler/dep_proxy (LoadError)``` as before.

I have seen issues where they recommand upgrading bundler in the CI before using it.
Some just remove a random thing like rake and it works by magic.

I don't see anything in the Circle CI of tootsuite very related to bundler, maybe the ruby image used does have a newer bundler ?

**Pull request merged**